### ISSUE TITLE
Import m3u files / add groups / group filters 

### ISSUE THREAD

**Issue by FrankBKK** [jellyfin/jellyfin-plugin-iptv#15]
It would be great if there is a way to import m3u files - adding several hundred stations manually is tedious ! 

Also would be great to be able to create groups (or create them on importing m3u) and filter by such a group

**Comment by PrplHaz4** [comment]
> It would be great if there is a way to import m3u files - adding several hundred stations manually is tedious !
> 
> Also would be great to be able to create groups (or create them on importing m3u) and filter by such a group

The live TV feature allows you to import m3u as a list of channels.

**Comment by FrankBKK** [comment]
yes, but I refer to the IPTV plugin which I prefer a lot over the Live TV setup
